{
  "id": "CVE-2025-28413",
  "sourceIdentifier": "cve@mitre.org",
  "published": "2025-04-07T16:15:25.543",
  "lastModified": "2025-04-08T16:15:25.983",
  "vulnStatus": "Received",
  "descriptions": [
    {
      "lang": "en",
      "value": "An issue in RUoYi v.4.8.0 allows a remote attacker to escalate privileges via the SysDictTypeController component"
    },
    {
      "lang": "es",
      "value": "Un problema en RUoYi v.4.8.0 permite que un atacante remoto escale privilegios a trav√©s del componente SysDictTypeController"
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
          "attackVector": "NETWORK",
          "attackComplexity": "LOW",
          "privilegesRequired": "NONE",
          "userInteraction": "NONE",
          "scope": "UNCHANGED",
          "confidentialityImpact": "HIGH",
          "integrityImpact": "HIGH",
          "availabilityImpact": "HIGH",
          "baseScore": 9.8,
          "baseSeverity": "CRITICAL"
        },
        "exploitabilityScore": 3.9,
        "impactScore": 5.9
      }
    ]
  },
  "weaknesses": [
    {
      "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
      "type": "Secondary",
      "description": [
        {
          "lang": "en",
          "value": "CWE-284"
        }
      ]
    }
  ],
  "references": [
    {
      "url": "https://github.com/20210607/cve_public/blob/main/ruoyi_case/CVE-2025-28413.md",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://github.com/yangzongzhuan/RuoYi",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://github.com/20210607/cve_public/blob/main/ruoyi_case/CVE-2025-28413.md",
      "source": "134c704f-9b21-4f2e-91b3-4a467353bcc0"
    }
  ]
}